lib/elixir/lib/calendar/date_range.ex

```diff
@@ -53,11 +53,7 @@ defmodule Date.Range do
     end
 
     def slice(range) do
-      %{
-        first_in_iso_days: first,
-        last_in_iso_days: last,
-        first: %{calendar: calendar}
-      } = range
+      %{first_in_iso_days: first, last_in_iso_days: last, first: %{calendar: calendar}} = range
```

Comment:
Line is too long, inconsistent with the other patterns on the same file.

Reply:
Same with maps as kwlists, I'd opt for always being multi-line (with trailing comas) if more than, say, one element (or even with any amount of elements, only 0 elements would be 1-line).